ansible/roles/ocp4-workload-aws-jumpbox/tasks/workload.yml

```diff
@@ -49,11 +49,11 @@
     mode: 0755
     owner: "{{ ansible_user }}"
   loop:
-  - { src: "./templates/create_jumpbox.j2", dest: "{{ aws_jumpbox_home }}/create_jumpbox.sh" }
-  - { src: "./templates/delete_jumpbox.j2", dest: "{{ aws_jumpbox_home }}/delete_jumpbox.sh" }
+  - {src: "./templates/create_jumpbox.j2", dest: "{{ aws_jumpbox_home }}/create_jumpbox.sh"}
+  - {src: "./templates/delete_jumpbox.j2", dest: "{{ aws_jumpbox_home }}/delete_jumpbox.sh"}
```

Comment:
This loop statement should not use the JSON notation. Also the prefix on the `src` items of `./templates/` is unnecessary. We should switch it to:

```
  loop:
  - src: create_jumpbox.j2
    dest: "{{ aws_jumpbox_home }}/create_jumpbox.sh
  - src: delete_jumpbox.j2
    dest: "{{ aws_jumpbox_home }}/delete_jumpbox.sh
```